Client/Client.Qt/Widgets/MessageWidget.cpp

```diff
@@ -1,176 +1,85 @@
 ﻿#include "MessageWidget.hpp"
 
+#include <QPainter>
+#include <QtEvents>
+
+#include "ChatHistory.hpp"
 #include "Style/Style.hpp"
 
-MessageWidget::MessageWidget(QString textMessage, QString nameOfUser, QListWidgetItem* Item,
-                             bool deletedMessage, QWidget* parent)
-    : QWidget(parent),
-      messageText(std::move(textMessage)),
-      userName(std::move(nameOfUser)),
-      dateTimeMessage(QDateTime::currentDateTime())
+MessageWidget::MessageWidget(QWidget* history, QString message, qint64 utc, QString user, const Style::MessageWidget& st)
```

Comment:
if we use std::move, why in parameters not r-value objects?

Reply:
Clang-Tidy says its better